crates/ruma-events/src/room/message.rs

```diff
@@ -425,13 +425,18 @@ impl EmoteMessageEventContent {
     }
 
     /// A convenience constructor to create a markdown emote.
+    ///
+    /// Returns an html emote message if some markdown formatting was detected, otherwise returns a
+    /// plain-text emote.
     #[cfg(feature = "markdown")]
     #[cfg_attr(docsrs, doc(cfg(feature = "markdown")))]
     pub fn markdown(body: impl Into<String>) -> Self {
         let body = body.into();
```

Comment:
With `FormattedBody::markdown` also having an `impl Into<String>` parameter type, this should not be needed anymore.

Reply:
The problem is that I need `body` twice, so I have to make sure that I can either clone or borrow it and I thought it would be simpler if I turn it into a `String` right away.